Source/Modules/xml.cxx

```diff
@@ -146,7 +146,10 @@ class XML:public Language {
 	Xml_print_parmlist(Getattr(obj, k));
       } else if (Cmp(k, "catchlist") == 0) {
 	Xml_print_parmlist(Getattr(obj, k), "catchlist");
-      } else {
+      } else if (Cmp(k, "templateparms") == 0) {
```

Comment:
I had to write it like this:

`Xml_print_parmlist(Getattr(obj, k), Char(k));`

print_parmlist takes a char * as the second argument so k needed to be converted before it could be passed. Thank you for the combining suggestion.